src/openrct2/actions/StaffSetNameAction.cpp

```diff
@@ -26,6 +26,12 @@ StaffSetNameAction::StaffSetNameAction(uint16_t spriteIndex, const std::string&
 {
 }
 
+void StaffSetNameAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("spriteIndex", _spriteIndex);
```

Comment:
I think just `id` would be better than `spriteIndex`. We no longer refer to entities as sprites.

Reply:
In `GuestSetNameAction` we used `peep`

Reply:
I just selected spriteIndex to line up with the member name, would it be preferred to have a more human readable name like "peep" or "peepId" for the outward facing API?

Reply:
I think `peep` would be a best. There are at least 2 other actions that use that.